frontend/src/pages/layout/Layout.tsx

```diff
@@ -4,8 +4,18 @@ import salesLogo from "../../img/logo.png";
 import github from "../../assets/github.svg";
 
 import styles from "./Layout.module.css";
+import { ChatHistoryButton } from "../../components/ChatHistoryButton/ChatHistoryButton";
+import { useApp } from "../../providers/AppProviders";
```

Comment:
@KenyerRamirez wouldn't be more meaningful a name more descriptive of what  actually does like `useAppContext` or something similar